backstop/all.json

```diff
@@ -20,6 +20,8 @@
   {"label": "CuratedListLinksDocs", "url": "/lists/qag-curatedlist"},
   {"label": "CuratedListPeopleContact", "url": "/lists/qag-curated-list2-people"},
   {"label": "DecisionMandate", "url": "/mandate/qag-decisionmandate", "screens": ["desktop", "mobile"]},
+  {"label": "DecisionTree", "url": "/decision-tree/qag-decisiontree", "screens": ["desktop", "mobile"]},
+  {"label": "DecisionTreeNested", "url": "/decision-tree/qa-decision-tree#s=125361&p=125361", "screens": ["desktop", "mobile"]},
```

Comment:
```suggestion
  {"label": "DecisionTreeNested", "url": "/decision-tree/qa-decision-tree", "screens": ["desktop", "mobile"]},
```

Reply:
I see, this is a branch page. Can you name it DecisionTreeBranch instead, to match the content type name